images/cross-cloud/sle-hpc/ondemand/profiles.yaml

```diff
@@ -23,4 +23,5 @@ profiles:
     description: GCE configuration
     include:
       - csp/gce/addon/gce-tools
+      - csp/gce/ondemand
```

Comment:
This line isn't necessary because the line `csp/gce/ondemand/hpc` below already includes all parent directories. It doesn't harm either, and may be used to create more descriptive image definitions I suppose. So far I've always opted for the most compact definition though.